src/client/components/number-filter-menu/number-filter-menu.tsx

```diff
@@ -38,6 +38,21 @@ function stringToNumberOrAny(startInput: string): number {
   return isNaN(parse) ? ANY_VALUE : parse;
 }
 
+const filterOptions: FilterOption[] = [
+  {
+    label: STRINGS.include,
+    value: Filter.INCLUDED,
+    svg: require('../../icons/filter-include.svg'),
+    checkType: 'check'
+  },
+  {
+    label: STRINGS.exclude,
+    value: Filter.EXCLUDED,
+    svg: require('../../icons/filter-exclude.svg'),
+    checkType: 'cross'
+  }
+];
```

Comment:
rather than recreating these you could do: `FilterOptionsDropdown.FILTER_OPTIONS.filter(option => option.value === Filter.INCLUDED || option.value === Filter.EXCLUDED`

or better yet something like:

`FilterOptionsDropdown.getFilterOptions(Filter.INCLUDED, Filter.EXCLUDED)`

:-p